libs/sdk-ui-ext/src/internal/tests/mocks/referencePointMocks.ts

```diff
@@ -13,6 +13,12 @@ import {
 import { OverTimeComparisonTypes } from "@gooddata/sdk-ui";
 import { ColumnWidthItem } from "@gooddata/sdk-ui-pivot";
 
+export const dateDatasetRef = {
```

Comment:
Can we use some item from `@gooddata/reference-workspace`? Or what is the type of this object?

Reply:
Yes, we can. Good point, it is `ObjRef` type.

Reply:
Fixed here: https://github.com/gooddata/gooddata-ui-sdk/pull/982/commits/91471d35bb7cf4b4692bd23d58f5a7b99b1eb663#diff-5f08587c92bcd0f9ddb19f08f914bf59571d2cb66c30259ca3743e9b5ec7d511R17. You can check it. :)